setup.py

```diff
@@ -214,10 +214,39 @@ def finalize_options(self):
 setup(
     name="shapely",
     version=version,
+    description="Manipulation and analysis of geometric objects",
+    # description="GEOS wrapped in numpy ufuncs",
+    # description="Geometric objects, predicates, and operations",
```

Comment:
The two descriptions commented out are from pygeos and shapely, respectively. The one that I used now (not commented out) is the one from the github repo's tagline here. Any preferences? (or alternatives?)

Reply:
I like the one you selected 👍